### PR TITLE
Add kubernetes deployment manifest

### PR DESCRIPTION
**Pull request by ankurcha**
* Added a kube deployment manifest.

### CODE REVIEW

File: kube-manifest.sample.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: zipkin-stackdriver
+        spec:
+          containers:
+            - name: zipkin-stackdriver
+              image: GoogleCloudPlatform/stackdriver-zipkin:latest
```
Comment (kevinmdavis) [inline kube-manifest.sample.yaml]: I've hosted the docker image at b.gcr.io/stackdriver-trace-docker/zipkin-collector. We can reference that here.

File: kube-manifest.sample.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+              imagePullPolicy: Always
+              env:
+                - name: PROJECT_ID
+                  value: my_project_id
+                - name: STORAGE_TYPE
+                  value: stackdriver
```
Comment (kevinmdavis) [inline kube-manifest.sample.yaml]: This is the default value for our zipkin collector, so this isn't required.

Comment (ankurcha) [inline kube-manifest.sample.yaml]: updated

File: kube-manifest.sample.yaml (hunk 1 above)
Comment (ankurcha) [inline kube-manifest.sample.yaml]: updated

**Pull request merged**